packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -115,54 +104,59 @@ export const RemoteDataTable = <T extends Record<string, unknown>>({
       />
     </Box>
   ) : (
-    <TableContainer sx={{ marginBottom: 100 }}>
+    <TableContainer>
       <MuiTable stickyHeader {...getTableProps()}>
         <TableHead>
           {headerGroups.map(({ getHeaderGroupProps, headers }) => (
             <TableRow {...getHeaderGroupProps()}>
-              {headers.map(column => (
-                <TableCell
-                  {...column.getHeaderProps(column.getSortByToggleProps())}
-                  sx={{
-                    backgroundColor: 'transparent',
-                    ...theme.typography.th,
-                  }}
-                >
-                  <Grid container>
-                    {column.render('Header')}
-                    {renderSortIcon(column)}
-                  </Grid>
-                </TableCell>
-              ))}
+              {headers.map(column => {
+                return (
+                  <TableCell
+                    {...column.getHeaderProps(column.getSortByToggleProps())}
+                    align={column.align}
+                    padding={column.id === 'selection' ? 'checkbox' : 'normal'}
+                    sx={{
+                      backgroundColor: 'transparent',
+                    }}
+                    sortDirection={
+                      column.isSorted
+                        ? column.isSortedDesc
+                          ? 'desc'
+                          : 'asc'
+                        : false
+                    }
+                  >
+                    <TableSortLabel
+                      hideSortIcon={column.id === 'selection'}
+                      active={column.isSorted}
+                      direction={column.isSortedDesc ? 'desc' : 'asc'}
+                      IconComponent={SortDesc}
+                    >
+                      {column.render('Header')}
+                    </TableSortLabel>
+                  </TableCell>
```

Comment:
Changes here were to try and be able to right align the checkbox header.

I went searching through the mui docs table examples and found this `TableSortLabel` component which flips the icon n stuff.

the `sortDirection` on the `TableCell` is apparently for blind peeps. 

will make an issue to encapsulate this elsewhere

Reply:
wicked.
I see that the icon has a tooltip 'toggle SortBy' which isn't super informative. not sure if that can be overridden